scripts/inference/cropland_mapping.py

```diff
@@ -27,12 +25,20 @@ def extract(self, image):
     parser.add_argument("miny", type=float, help="Minimum Y coordinate (south)")
     parser.add_argument("maxx", type=float, help="Maximum X coordinate (east)")
     parser.add_argument("maxy", type=float, help="Maximum Y coordinate (north)")
+    parser.add_argument(
+        "--epsg",
```

Comment:
What does this argument do exactly? Is this for output EPSG? Internally there is still the datacube epsg which is used to feed lat/lon to Presto?

Reply:
No this is the epsg for the input bounds minx, miny, maxx, maxy

I modified the description to the parameter to clarify this